public/css/style.css

```diff
@@ -41,3 +41,8 @@ body {
 .event-img {
     width: 18rem;
 }
+
+label.required:after {
```

Comment:
@daria-sieroshtan 
It is very good that you lerned this feature yourself, congrats!
Though manipulating content with CSS is generally considered to be a bad practise. In this particular case you should do it by modifying Twiggy form theme

Reply:
I found this code in Symfony documentation.
I will change this is case of need for Event form, otherwise I prefer this simple solution.

Reply:
OK